eknrenderer/eknrenderer/eknr-boxed-set-info.c

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2018 Endless Mobile, Inc. */
```

Comment:
Probably needs an actual copyright header, but I'm just following the example in ekncontent for now.